Form-Controls/styles.css

```diff
@@ -0,0 +1,288 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,300&display=swap');
+
+/* root */
+
+:root {
+    --blue: #1cc5e7;
+    --purple: #b49df8;
+    --pink: #f17ed2;
+    --blue-green: #15c395;
+    --gray: #aea3a6;
+    --green: #73f777;
+    --yellow: #f5f885; 
+    --aqua: #1cc5e7;
+
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing:border-box;
+}
+
+body {
+    color: black;
+    font-family: ''Montserrat', sans-serif';
```

Comment:
Is this being picked up by the page? Look for a syntax error in this CSS property